### PR TITLE
update check links

### PR DESCRIPTION
**Pull request by GeorgeGoodall**
Ticket: https://github.com/orgs/digital-land/projects/7/views/24?filterQuery=assignee%3A%40me+-status%3ABacklog%2CUnprioritised%2C%22In+Testing%22%2C%22Ready+for+Dev%22%2C%22Scoping+for+Dev%22%2C%22Creating+Analysis%22%2C%22User+Research%2FTesting%22%2C%22In+design%22++sprint%3A%40current&pane=issue&itemId=74896459

this PR handles the first part of this ticket which is updating the subpaths for the site.

I'm not sure if we should merge this in before chris c and ernest return from holiday and can setup redirect links?

### CODE REVIEW

**Comment by GeorgeGoodall** [comment]
the acceptance tests are currently failing for this as they are written for this branch but are tested against the main branch (using the production site) for which the urls are different

File: test/integration/pages_load_ok.playwright.test.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ test.describe('with a valid session, the user can access the later form pages',
     await startPage.navigateHere()
     await startPage.clickStartNow()
 
-    await checkRouteResponse(page, '/dataset', [200, 304])
+    await checkRouteResponse(page, '/check/dataset', [200, 304])
   })
 
   test('/geometry-type', async ({ page }) => {
```
Comment (rosado) [inline test/integration/pages_load_ok.playwright.test.js:58]: Missing update to the test descriptions (like in the tests above).

Looks OK, otherwise.

**Review by rosado — CHANGES_REQUESTED** [review]

**Review by rosado — DISMISSED** [review]

File: test/integration/pages_load_ok.playwright.test.js (hunk 1 above)
Comment (DilwoarH) [reply test/integration/pages_load_ok.playwright.test.js:58]:
```suggestion
  test('/check/geometry-type', async ({ page }) => {
```

File: test/integration/pages_load_ok.playwright.test.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ test.describe('with a valid session, the user can access the later form pages',
     await datasetPage.selectDataset(datasets.Tree)
     await datasetPage.clickContinue()
 
-    await checkRouteResponse(page, '/geometry-type', [200, 304])
+    await checkRouteResponse(page, '/check/geometry-type', [200, 304])
   })
 
   test('/upload-method', async ({ page }) => {
```
Comment (DilwoarH) [inline test/integration/pages_load_ok.playwright.test.js:71]:
```suggestion
  test('/check/upload-method', async ({ page }) => {
```

File: test/integration/pages_load_ok.playwright.test.js [hunk 3]
```diff
@@ -82,7 +82,7 @@ test.describe('with a valid session, the user can access the later form pages',
     await geometryTypePage.selectGeometryType(geometryTypes.point)
     await geometryTypePage.clickContinue()
 
-    await checkRouteResponse(page, '/upload-method', [200, 304])
+    await checkRouteResponse(page, '/check/upload-method', [200, 304])
   })
 
   test('/upload', async ({ page }) => {
```
Comment (DilwoarH) [inline test/integration/pages_load_ok.playwright.test.js:88]:
```suggestion
  test('/check/upload', async ({ page }) => {
```

File: test/integration/pages_load_ok.playwright.test.js [hunk 4]
```diff
@@ -103,7 +103,7 @@ test.describe('with a valid session, the user can access the later form pages',
     await uploadMethodPage.selectUploadMethod(uploadMethods.File)
     await uploadMethodPage.clickContinue()
 
-    await checkRouteResponse(page, '/upload', [200, 304])
+    await checkRouteResponse(page, '/check/upload', [200, 304])
   })
 
   test('/url', async ({ page }) => {
```
Comment (DilwoarH) [inline test/integration/pages_load_ok.playwright.test.js:109]:
```suggestion
  test('/check/url', async ({ page }) => {
```

**Review by DilwoarH — CHANGES_REQUESTED** [review]
Adding to the comment from @rosado

**Review by rosado — APPROVED** [review]

**Pull request merged**